cgdrush

```diff
@@ -0,0 +1,14 @@
+#/bin/bash
+
+export SCRIPT_NAME=drush 
+CGDRUSH=$( readlink -f "$0" )
+SCRIPT_PATH=$( dirname "$CGDRUSH" )
+export SCRIPT_FILENAME=$SCRIPT_PATH/drush.php 
+export REQUEST_METHOD=GET 
+export args="drush" 
+for i in $* 
+  do
+  arg=$(urlencode $i)
+  args="$args&$arg"
```

Comment:
How does `args` get passed to php-fpm?

Reply:
shell script encode them from command line and store them to args env var then php get them in $_SERVER['args'] and decode line 58 to 65.

Reply:
in case you're wondering the problem is send array to php-fpm for argv